src/transactions/FeeBumpTransactionFrame.h

```diff
@@ -91,11 +84,50 @@ class FeeBumpTransactionFrame : public TransactionFrameBase
     insertKeysForFeeProcessing(UnorderedSet<LedgerKey>& keys) const override;
     void insertKeysForTxApply(UnorderedSet<LedgerKey>& keys) const override;
 
-    void processFeeSeqNum(AbstractLedgerTxn& ltx, int64_t baseFee) override;
-
+    void processFeeSeqNum(AbstractLedgerTxn& ltx, int64_t baseFee,
+                          TransactionResult& txResult) override;
     StellarMessage toStellarMessage() const override;
 
     static TransactionEnvelope
     convertInnerTxToV1(TransactionEnvelope const& envelope);
+
+#ifdef BUILD_TESTS
```

Comment:
note: you don't need any of this we directly use `TestTransactionFrame` in test code